layouts/index.html

```
@@ -1,60 +1,128 @@
+</div>
+{{ end }}
+
+{{- /* Compact widgets */ -}}
+<div class="widgets">
+    {{- /* Reading widget */ -}}
+    {{ $books := first 3 (where .Site.RegularPages "Section" "books") }}
+    {{ if $books }}
+    <div class="widget">
+        <div class="widget-header">
+            <h3 class="section-label">Reading</h3>
+            <a href="{{ "/media/books" | relLangURL }}" class="section-more">More →</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)